CHANGELOG

```diff
@@ -1,5 +1,9 @@
 == Version 4.6.0
```

Comment:
It sure should have been. I got impatient to release the new gem version and merged without review, figuring this small change wouldn't need one. Shows that even the smallest changes should be properly reviewed 😄 

Fixing it in https://github.com/Shopify/shopify_api/pull/356